src/main/java/frc/robot/controllers/BaseDriveController.java

```diff
@@ -27,6 +27,10 @@ public abstract class BaseDriveController {
      */
     public abstract boolean getSwerveRelative();
 
+    public boolean getHeadingLock() {
```

Comment:
Is this not an abstract method just so `TwistJoystickController` doesn't have to implement it?